mmv1/templates/terraform/constants/router_bgp_peer.erb

```diff
@@ -0,0 +1,14 @@
+func ipv6RepresentationDiffSuppress(_, old, new string, d *schema.ResourceData) bool {
+    //Diff suppress any equal IPV6 address in different representations
+    //An IPV6 address can have long or short representations
+    //E.g 2001:0cb0:0000:0000:0fc0:0000:0000:0abc, after compression:
+    //A) 2001:0cb0::0fc0:0000:0000:0abc (Omit groups of all zeros)
+    //B) 2001:cb0:0:0:fc0::abc (Omit leading zeros)
+    //C) 2001:cb0::fc0:0:0:abc (Combining A and B)
+    //The GCP API follows rule B) for normalzation
+
+    oldIp := net.ParseIP(old)
+    newIp := net.ParseIP(new)
+    log.Printf("[DEBUG] Old IP is %s, New IP is %s", oldIp, newIp)
```

Comment:
```suggestion
```
We probably can remove this line. I believe this is for debug only?

Reply:
Yes, it is only for the debugging purpose. I'll remove the line.